### ISSUE TITLE
Cannot add multiple "Notification Area"s

### ISSUE THREAD

**Issue by davidhedlund** [mate-desktop/mate-panel#1399]
#### Expected behaviour
Make it possible to add more "Notifcation Area"s because it's possible to add multiple entries of the rest of the items in "Add to Panel".

#### Actual behaviour
More than one "Notification Area" cannot be added, but still the "Add" button indicates that it could be added.

#### Steps to reproduce the behaviour
![image](https://github.com/mate-desktop/mate-panel/assets/6416073/62848b51-0a2e-4464-8a1b-c7a922a41c3a)
Tried to add more than one "Notification Area"

#### MATE general version
1.26.0

#### Package version
1.26.0

#### Linux Distribution
Ubuntu MATE 23.04

**Comment by rilian-la-te** [comment]
It cannot be possible, if we using XEMBED at least.

**Comment by davidhedlund** [comment]
Then the "Add" button should be disabled for the "Notification Area" if it has already been added, right?

**Comment by davidhedlund** [comment]
> It cannot be possible, if we using XEMBED at least.

Then the "Add" button should be disabled for the "Notification Area" if it has already been added, right?

**Comment by lukefromdc** [comment]
With in-process builds of the notification area applet, you can indeed load multiple instances of it, though only SNI icons (which do not use Xembed) will appear on it. That is essentially the same behavior as on wayland, in which case any number of instances of the applet should indeed work, but traditional tray icons (which use Xembed) are excluded.

**Comment by davidhedlund** [comment]
Thank you for your technical feedback.

> traditional tray icons (which use Xembed) are excluded.

Can they be included?

**Comment by lukefromdc** [comment]
I do not know enough about xembed to answer that one.

**Comment by rilian-la-te** [comment]
No, XEMBED can be active only for one instance per X screen. For SNI, as @lukefromdc said, it is possible to show multiple tray instances. I tested SNI with both sntray applet and traditional MATE SNI, and both works together on one panel.
